Tests/SwiftParserTest/translated/ConsecutiveStatementsTests.swift

```diff
@@ -142,12 +142,14 @@ final class ConsecutiveStatementsTests: XCTestCase {
       }
       """,
       diagnostics: [
-        DiagnosticSpec(message: "consecutive declarations on a line must be separated by ';'", fixIts: ["insert ';'"])
+        DiagnosticSpec(message: "consecutive declarations on a line must be separated by ';'", fixIts: ["insert newline", "insert ';'"])
       ],
+      applyFixIts: ["insert newline"],
       fixedSource: """
         class C {
           // In a sequence of declarations.
-          var a, b : Int; func d() -> Int {}
+          var a, b : Int
+        func d() -> Int {}
```

Comment:
😬 We should  pick up the indentation of the current line. I think you should be able to just walk the tokens backwards until you find a token that contains indentation. The functions in https://github.com/apple/swift-syntax/blob/main/Sources/SwiftBasicFormat/Trivia%2BFormatExtensions.swift might be useful for that. It seems like they might be generally useful, so you could make them public.